### PR TITLE
refactor(process): make use of the ECL toolkit - INNO-403

### PR DESCRIPTION
**Pull request by yhuard**
Please ensure that it works the same as earlier

### CODE REVIEW

**Comment by emeryro** [comment]
I confirm that watchers now works correctly on Windows (we have to be sure that is still works on other OS...)
For now, it just triggers linter but doesn't re-build scss files

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "start": "npm-run-all clean build --parallel serve watch",
     "test:coding-conventions": "npm run lint",
     "test:functional": "wdio test/wdio.conf.js",
-    "watch:scripts": "chokidar ./framework/**/*.js -c 'eslint {path} && npm run build:scripts'",
-    "watch:styles": "chokidar ./framework/**/*.scss -c 'stylelint {path} || true && npm run build:styles'",
+    "watch:scripts": "chokidar ./framework/**/*.js -c \"eslint {path} && npm run build:scripts\"",
+    "watch:styles": "chokidar ./framework/**/*.scss -c \"stylelint {path} || true && npm run build:styles\"",
```
Comment (emeryro) [inline package.json]: "|| true" has to be removed

Comment (yhuard) [inline package.json]: I will do it

**Comment by yhuard** [comment]
@emeryro it's now fixed

**Comment by yhuard** [comment]
For me it's ready, please review it one last time @degliwe @emeryro :)

File: framework/components/feedback-forms/package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
```
Comment (degliwe) [inline framework/components/feedback-forms/package.json]: doesn't appear in the component library

File: framework/components/forms/controls/package.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
```
Comment (degliwe) [inline framework/components/forms/controls/package.json]: doesn't appear in the component library

File: framework/components/site-menus/package.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
 {
```
Comment (degliwe) [inline framework/components/site-menus/package.json]: doesn't appear in the component library

File: framework/layout/blocks/package.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
 {
```
Comment (degliwe) [inline framework/layout/blocks/package.json]: doesn't appear in the component library

File: framework/utilities/clearfix/package.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (degliwe) [inline framework/utilities/clearfix/package.json]: doesn't appear in the component library

File: framework/utilities/flex/package.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (degliwe) [inline framework/utilities/flex/package.json]: doesn't appear in the component library

File: framework/utilities/screen-reader/package.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (degliwe) [inline framework/utilities/screen-reader/package.json]: doesn't appear in the component library

**Pull request merged**